### DISCUSSION TITLE
New 'outline' keyword in v3 should be documented in upgrade guide

### DISCUSSION THREAD

**Discussion by therealgilles** [tailwindlabs/tailwindcss#7028] [General]
It would be great if the upgrade guide could mention that 'outline' is now a keyword used by tailwindcss. I had used it as a regular class name before and discovered it is now in conflict.

**Comment by simonswiss** [comment]
Hey! Do you mean the `outline-*` utilities?

Technically there was already `outline-white` and `outline-black` in version 2. In the upgrade guide, we mention how to bring back the same styles if you were using those:

https://tailwindcss.com/docs/upgrade-guide#outline-black-white

Please let me know if I didn't interpret the question properly!
